### PR TITLE
Updates from lantern

### PR DESCRIPTION
**Pull request by oxtoacart**

### CODE REVIEW

File: osversion_android.go [hunk 1]
```diff
@@ -1,87 +1,11 @@
 package osversion
 
-/*
```
Comment (xiam) [inline osversion_android.go]: Don't we need versions on android?

Comment (oxtoacart) [inline osversion_android.go]: Yes, I think so.  This looks like a bug (possibly one that I introduced).

Comment (myleshorton) [inline osversion_android.go]: We're using this code in production right @oxtoacart? @atavism do we need the osversion on Android? Is that used?

Comment (oxtoacart) [inline osversion_android.go]: Yes, as far as I know we're using this in production.

**Pull request closed without merging**